modules/remotebackend/test-remotebackend.cc

```diff
@@ -339,4 +339,15 @@ BOOST_AUTO_TEST_CASE(test_method_directBackendCmd) {
    BOOST_CHECK_EQUAL(be->directBackendCmd("PING 1234"), "PING 1234");
 }
 
+BOOST_AUTO_TEST_CASE(test_method_lookup) {
```

Comment:
This test is specific to remotebackend. Please add a test or two to pdns/test-dnsrecords_cc.cc instead.